polynote-frontend/style/styles.less

```diff
@@ -304,12 +310,13 @@ body {
       bottom: 0;
       width: @cell-margin-width;
       background: @ui-background;
-      border: 1px solid @ui-border;
+      border-top-left-radius: 10px;
+      box-shadow: -1px 1px 2px rgba(0, 0, 0, 0.25);
```

Comment:
Also not a fan of the box shadow. Maybe there's a different way we could add contrast with the background, if that's what's needed?